src/cargo/util/toml/targets.rs

```diff
@@ -106,13 +105,10 @@ fn clean_lib(toml_lib: Option<&TomlLibTarget>,
             let legacy_path = package_root.join("src").join(format!("{}.rs", lib.name()));
             if legacy_path.exists() {
                 {
-                    let short_path = without_prefix(&legacy_path, package_root)
-                        .unwrap_or(&legacy_path);
-
                     warnings.push(format!(
                         "path `{}` was erroneously implicitly accepted for library `{}`,\n\
                          please rename the file to `src/lib.rs` or set lib.path in Cargo.toml",
-                        short_path.display(), lib.name()
+                        legacy_path.display(), lib.name()
                     ));
                 }
```

Comment:
uuuh, i just did that but also applied `rustfmt` :)) that was a bad idea as it formatted a lot of unrelated code in this file!(no worries, I can undo)

but for some reason my vim indenting isn't aligning the `));` to the same indent as `warnings.push`

Just wondering, any reason why `rustfmt` wasn't used ?